SAO deleted tables (#8458)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/state-aware-about.md

To prevent a job from being built too frequently even when the code or data state has changed, you can reduce build frequency by using the `build_after` config. For information on how to use `build_after`, refer to [Model freshness](/reference/resource-configs/freshness) and [Advanced configurations](/docs/deploy/state-aware-setup#advanced-configurations).

### Handling deleted tables

> State-aware orchestration detects and rebuilds models when a model's table is deleted in the warehouse, even if there are no code or data changes.

Reviewer: ```suggestion
State-aware orchestration detects and rebuilds models when their tables are deleted in the warehouse, even if there are no code or data changes.
```
